resources/qml/SettingsWidget.qml

```diff
@@ -39,10 +39,9 @@ Item {
                 color: RadianceStyle.mainTextColor
             }
 
-            ComboBox {
-                id: modelNameComboBox
-                editable: true
-                model: ["gui", "cli"]
+            TextInput {
+                id: modelNameTextInput
+                text: "model.json"
```

Comment:
What's the reasoning behind this change?

Reply:
I don't think there's ever a reason you would want to load the CLI model into radiance, and it felt silly to have a combobox with a single thing in it.

Reply:
You would want to load it, edit it, and save it.